### ISSUE TITLE
load/save rete-network --load/save is redundant

### ISSUE THREAD

**Issue by marinier** [SoarGroup/Soar#311]
These commands seem redundant with themselves:

    load rete-network --load <filename>
    save rete-network --save <filename>

It seems unnecessary to specify load and save twice -- probably this is a holdover from the old `rete-net` command. It would be cleaner (and more consistent with the other load/save subcommands) to just do:

    load rete-network <filename>
    save rete-network <filename>

**Comment by mazina** [comment]
You're completely right about the reasoning.  There are several commands that have weird things like that.   The main reason is that we didn't have the resources to do re-do all of Soar's commands so that we standardized the syntax of each command's parameters, especially since doing that would also mean we'd need to also add a lot more code to maintain backwards compatibility.  Instead, we made it so that all of the sub-commands have the same parameters that they used to have as top-level commands.  Doing that allowed us to get most of the backwards compatibility by using aliases from the old commands to the new commands.  (Soar still has to translate some parameters and commands manually, for example most of the old learn command.)
